java/compaction/compaction-job-creation/src/main/java/sleeper/compaction/job/creation/CreateJobs.java

```diff
@@ -143,6 +161,46 @@ public void createJobsForTable(TableProperties tableProperties) throws StateStor
         }
     }
 
+    private void createJobsFromLeftoverFiles(TableProperties tableProperties, List<FileInfo> activeFileInfosWithNoJobId,
+                                             List<Partition> allPartitions, List<CompactionJob> compactionJobs) {
```

Comment:
Can you add a log line here to say that it is creating compaction jobs from all files?